reference/price_lists.v3.yml

```diff
@@ -2668,6 +2668,31 @@ paths:
               schema:
                 $ref: '#/components/schemas/AssignmentsForGetResponse'
     post:
+      tags:
+        - Price Lists Assignments
+      description: Creates a batch of `Price List Assignments`.
+      summary: Create Price List Assignments
+      operationId: CreatePriceListAssignments
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/CreateBatchPriceListAssignmentsRequest"
+        required: true
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/SuccessBatchResponse"
+        "422":
+          description: Error response. Includes the errors for each reference ID.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/PriceListAssignmentsBatchErrorResponse"
```

Comment:
This ref's schema doesn't match the format for the `422` I received:

```json
{
    "status": 422,
    "title": "Bulk Price List Assignment operation has failed",
    "type": "https://developer.bigcommerce.com/api-docs/getting-started/api-status-codes",
    "errors": {
        "0.price_list_id": "Request includes invalid price list id(s).",
        "0.customer_group_id": "The combination of customer_group_id 1 and channel_id 1 already assigned to price_list_id 4"
    },
    "meta": {
        "saved_records": 0
    }
}
```

Reply:
I changed it, see the screenshot. Not sure how detailed we can make it
![422 error](https://user-images.githubusercontent.com/96258747/202574676-ee75923b-4441-4974-9a10-8d99613866b8.png)

Reply:
This looks good, thanks for updating